talent/job-descriptions/offers/lead-applications.md

```diff
@@ -0,0 +1,37 @@
+Lead Software Engineer in the Applications team
+------------------------------------------
+
+### Location & Seniority
+
+**Location:** Madrid / Fully remote  
+**Seniority:** Lead 
+
+**Compensation:** 58,000 Euros + [stock options](https://github.com/src-d/guide/blob/master/talent/esop.md)
+
+### [Company](../company-section.md)
+
+### [Engineering at source{d}](../engineering-section.md)
+
+### Role
+
+This position is for a Lead Engineer in the Applications team.
+
+- As a Lead Engineer, you will be expected to plan current and future work, provide mentorship, overseeing progress of the team and, in general, be able to make day-to-day work smooth for everyone in the team.
+
+- In terms of coding, you will have a two-fold role: on one hand it's a Data Engineering, building production-level pipelines to collect and extract data from all the git repositories in the world. On the another hand it's applied Machine Learning, by productionizing Machine Learning research and delivering it as part of end-user applications.
```

Comment:
You should add the frontend part, and maybe the ML part it's not that important.

Reply:
Replacing ML clause with something closer web application would be more accurate and makes perfect sense.

Something a long the lines
`On the another hand it's building web applications to productionize Machine Learning research with on both, backend and frontend sides.`